### DISCUSSION TITLE
Docs typos and diction errors

### DISCUSSION THREAD

**Discussion by proofer** [nvim-mini/mini.nvim#967] [General]
My github moniker derives from my proofreading activities. The initial entry in this discussion:

[readmes/mini-clue.md](https://github.com/echasnovski/mini.nvim/blob/d7e05b30d842e543366ff5474076a0726f783a32/readmes/mini-clue.md?plain=1#L80) (2 instances); [doc/mini-clue.txt](https://github.com/echasnovski/mini.nvim/blob/d7e05b30d842e543366ff5474076a0726f783a32/doc/mini-clue.txt#L75) (3 instances); [doc/mini-clue.txt](https://github.com/echasnovski/mini.nvim/blob/d7e05b30d842e543366ff5474076a0726f783a32/doc/mini-clue.txt#L75) (same 3):
`full proof` -> `foolproof` -- "foolproof" is a single word meaning "incapable of going wrong or being misused".

**Comment by echasnovski** [comment]
Thanks, I learned something new today :)
I'll fix this soon.

**Edit**: Should be fixed on latest `main`.
